caches_bench_test.go

```diff
@@ -230,8 +238,8 @@ func BigCacheGet[T any](cs constructor[T], b *testing.B) {
 
 	hitCount := 0
 	for i := 0; i < b.N; i++ {
-		id := rand.Intn(maxEntryCount)
-		data, _ := cache.Get(key(id))
+		//id := rand.Intn(maxEntryCount)
```

Comment:
how about generating `id`s upfront ant then just reading them as `ids[i]`? That should be much faster